src/Opg/Core/Model/Entity/CaseItem/Deputyship/Order.php

```diff
@@ -21,10 +21,10 @@
 class Order extends Deputyship implements HasCourtFundsInterface
 {
     /**
-    * @ORM\Column(type = "string", nullable = true)
-    * @var string
-    * @Groups("api-task-list")
-    */
+     * @ORM\Column(type = "string", nullable = true)
```

Comment:
These wrongly indented asterisks were like a splinter in my eyes. so I fixed them, too.